src/ccalltest.c

```diff
@@ -26,7 +27,7 @@ int __declspec(noinline)
 #else
 int __attribute__((noinline))
 #endif
-testUcharX(unsigned char x) {
+DLLEXPORT testUcharX(unsigned char x) {
```

Comment:
CFLAGS has a special meaning inside of src/Makefile due to an override at the top